care.js

```diff
@@ -4,6 +4,7 @@ var twitterbot = require(__dirname + '/twitterbot.js');
 var gitbot = require(__dirname + '/gitbot.js');
 var pomodoro = require(__dirname + '/pomodoro.js');
 var ansiArt = require('ansi-art').default;
+var cowsay = require("cowsay");;
```

Comment:
nit: single quotes, not double, please. also, there's two `;`

Reply:
fixed